packages/headless/src/controllers/core/facets/facet/headless-core-facet.ts

```diff
@@ -88,13 +94,27 @@ export interface CoreFacet extends Controller {
    */
   toggleSelect(selection: FacetValue): void;
 
+  /**
+   * Toggles the exclusion the specified facet value.
+   *
+   * @param selection - The facet value to toggle exclusion.
+   */
+  toggleExclude(selection: FacetValue): void;
+
   /**
    * Toggles the specified facet value, deselecting others.
    *
    * @param selection - The facet value to toggle.
    */
   toggleSingleSelect(selection: FacetValue): void;
 
+  /**
+   * Excludes the specified facet value, including others.
```

Comment:
Hmm you're right. This was written when I thought that `includeAll` was going to be a thing. After some discussion, we agreed that it would make sense to instead `deselectAll` when this action is dispatched since it's functionally the same thing (e.g., reset the state to idle for all values except this one).

Perhaps I should just stick with `deselecting others.`?

Reply:
'reset the state to idle for all values except this one' sounds good if it can be used?
if not `while including|deselecting others` sounds good.

Reply:
I think I'l leave it as `while deselecting` others. Saying that it `includes` other values is a bit misleading as that isn't really a state that exists on the facet (e.g., `included` is functionally the same as `idle | selected`). "Deselecting" is exactly what happens under the hood of this controller, so I'll leave the description as such for clarity 🙂